### PR TITLE
Update error messages

### PR DESCRIPTION
**Pull request by o-lim**
Update error messages with "Error:" prefix and validate numeric command line options.

### CODE REVIEW

File: busted/modules/test_file_loader.lua [hunk 1]
```diff
@@ -70,6 +70,11 @@ return function(busted, loaders, options)
       end
     end
 
+    if #fileList == 0 then
+      local err = 'No test files found matching Lua pattern: ' .. pattern
```
Comment (ajacksified) [inline busted/modules/test_file_loader.lua]: Can we i18n this and pull it in from `say`?

**Comment by o-lim** [comment]
Sure, but I won't be able to fill in any of the other languages.

How is this?

**Comment by iggyvolz** [comment]
I can do French - I'll add that as a PR to your PR, if that makes any sense.

**Comment by ajacksified** [comment]
No need to fi ll in any of the other langs - just putting new strings into `say` is a good habit anyway. Probably should redo cliargs as a separate pr at some point.

**Comment by ajacksified** [comment]
@iggyvolz thanks!

**Comment by o-lim** [comment]
Merged french translation from @iggyvolz. Thanks.

**Comment by ajacksified** [comment]
:+1:

**Pull request merged**